components/molecule/stepper/src/Step/settings.js

```diff
@@ -26,6 +26,13 @@ export const getIcon = ({
 
 export const getLabel = ({label, steps, step, design, current}) => {
   if (design === DESIGN.DEFAULT) return label
-  else if (design === DESIGN.BASIC) return label
-  else if (design === DESIGN.COMPRESSED && current) return `${step}/${steps}${label ? `: ${label}` : ''}`
+  if (design === DESIGN.BASIC) return label
+  if (design === DESIGN.COMPRESSED && current)
+    return (
+      <>
+        {`${step}/${steps}`}
+        {label && `: `}
+        {label}
```

Comment:
🚨 🚨 🚨
Cannot do that because in case of using google translate addon it wraps every "" dom text with a.

Reply:
Thats why the complete text is composed as a single template string.

Reply:
sadly we can't return what we're currently returning... ie `3/5: [Object object]`

Reply:
why it has Object as a label?

Reply:
https://github.com/SUI-Components/sui-components/pull/2695#issuecomment-1930031816